src/main/java/mate/academy/internetshop/service/BucketService.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.internetshop.service;
+
+import java.util.List;
+
+import mate.academy.internetshop.model.Bucket;
+import mate.academy.internetshop.model.Item;
+
+public interface BucketService {
+    Bucket add(Bucket bucket);
+
+    Bucket get(Long id);
+
+    void delete(Long id);
+
+    Bucket addItem(Bucket bucket, Item item);
+
+    Bucket clear(Bucket bucket);
+
+    List getAllItems(Bucket bucket);
```

Comment:
```suggestion
    List<Item> getAllItems(Bucket bucket);
```
we should do the same with all methods which return list